tests/collections/asr/test_asr_context_biasing.py

```diff
@@ -0,0 +1,139 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import os
+import tempfile
+
+import numpy as np
+import pytest
+import torch
+from pytorch_lightning import Trainer
+
+from nemo.collections.asr.models import EncDecCTCModelBPE
+from nemo.collections.asr.parts import context_biasing
+from nemo.collections.asr.parts.context_biasing.ctc_based_word_spotter import WSHyp
+from nemo.collections.asr.parts.utils import rnnt_utils
+
+
+def conformer_ctc_bpe_model():
```

Comment:
Make a fixture with scope "module" to avoid reloading
Or you can move it to `conftest.py` to allow other tests to reuse it